lib/ansible/modules/system/iptables.py

```diff
@@ -460,6 +460,19 @@
     limit_burst: 20
     log_prefix: "IPTABLES:INFO: "
     log_level: info
+
+- name: Block container to visit 169.254.169.254
```

Comment:
It would be good if you could add more information to this example. Right now, it is hard to understand what it does and especially why it does that without the information you provided in the PR's description.

(Especially the link to https://blog.appsecco.com/an-ssrf-privileged-aws-keys-and-the-capital-one-breach-4c3c2cded3af would be helpful IMO.)

Reply:
The data breach of bank Wellsfargo is caused by the access to this aws meta data server 169.254.169.254, so we try to block all containers' access to this aws meta data server.

Sorry for delayed reply, just found the github email is spam